Automattic/jetpack
# VideoPress: check if uploaded video still exists before adding it

#### Changes proposed in this Pull Request:

* When adding video from local library that was previously uploaded to VideoPress, check if videopress attachment still exists

#### Other information:


#### Jetpack product discussion


p1661453915928049-slack-C02TQF5VAJD

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Upload a video to your local Media Library
* Using the block, upload it to VideoPress (choose it from the Media Library)
* Go to Media Library and delete the VideoPress video that was created when you uploaded it
* Add a new post again and try to add the video again
* Make sure the video is uploaded again
